### PR TITLE
EWL-7175: Contact Us Form - Enhancement

### PR DESCRIPTION
**Pull request by caharder**
…utton selection and removes Next button on first page when Javascript is enabled

## Ticket(s)

**Github Issue**
N/A

**Jira Ticket**
- [EWL-7175 - Contact Us Form - Enhancement](https://issues.ama-assn.org/projects/EWL/issues/EWL-7175)

## Description
Adds functionality to submit first page of Contact Us form on radio button selection and removes Next button on first page when Javascript is enabled

## To Test
- Pull branch EWL-7175-contact-us-form
- Browse to contact us page
- Verify that "Next" button is not visible
- Verify that form progresses to the next form page (with contact form) on selection of Physician, Resident, or Med Student
- Verify that form progresses to the next page (with AMA Automated Answer Center) on selection of Public/Other

## Visual Regressions

_Please provide the pull request ID below. This will then reference the automated VRT report after it has been generated._

A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/origin/EWL-7175-contact-us-form/html_report/index.html).

**Before proceeding:** Run visual regression tests locally to ensure new work does not introduce new bugs.

_The visual regression tests always compare against the production version of the style guide. Changes and new work will show up as test failures. Please describe your work so that the report can be reviewed by the team._

If you are creating or updating a pattern be sure you have created or updated the [scenario in `backstop.json`](ama-style-guide-2/styleguide/backstop.json).

_After changes in the report are documented and demonstrate your desired changes, please mark the pull request as `ready for review`.

For more information on visual regression testing review the [How do I run tests?](https://issues.ama-assn.org:8446/confluence/pages/viewpage.action?pageId=23298568) document in Confluence.

## Relevant Screenshots/GIFs
**Before**
contact_us_before

**After**
contact_us_after

## Remaining Tasks
N/A

## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Comment by designerneil** [comment]
@caharder the next button appears again when I click on one of the options then click back button.

File: styleguide/source/assets/js/webforms.js [hunk 1]
```diff
@@ -48,6 +48,12 @@
     }
   }
 
+  // Submits first page of Contact Us form on radio button selection
+  var $webform_buttons = $('#webform-submission-contact-us-add-form input[type="radio"]');
+  $webform_buttons.click(function(e) {
```
Comment (designerneil) [inline styleguide/source/assets/js/webforms.js]: I would bind the click.

`$( "#foo" ).bind( "click", function() {
  alert( "User clicked on 'foo.'" );
});`

File: styleguide/source/assets/scss/03-organisms/_contact-us-webform.scss [hunk 2]
```diff
@@ -39,3 +39,12 @@
     @include gutter($margin-bottom-full...);
   }
 }
+
+// Removes Next button on first page when Javascript is enabled
+html.js {
+  #webform-submission-contact-us-add-form {
```
Comment (designerneil) [inline styleguide/source/assets/scss/03-organisms/_contact-us-webform.scss]: probably should pass in `[type="submit"]` here

**Comment by caharder** [comment]
The IDs changed after clicking the back button so I changed the selectors to classes (which do not change). I also had to bind the click event to the buttons on ajax load since the binding was lost when moving between form pages.

**Comment by designerneil** [comment]
@caharder good job

**Review by designerneil — APPROVED** [review]

**Comment by designerneil** [comment]
Works as described. The next button is hidden and the form submits automatically.

**Pull request merged**